filter/usr/share/nethesis/NethServer/Template/Mail/Filter.php

```diff
@@ -64,18 +64,12 @@
     ->insert($view->textInput('BlockAttachmentCustomList', $view::LABEL_NONE))
 );
 
-//Retrieve the rspamd name and password
-$db = $view->getModule()->getPlatform()->getDatabase('configuration');
-$alias = $db->getProp('rspamd','alias');
-$password = $db->getProp('rspamd','password');
-
 //Retrieve the  rspamd URL
 $host = explode(':',$_SERVER['HTTP_HOST']);
```

Comment:
The $_SERVER['HTTP_HOST'] variable should include the port number. This line can be removed and the variable used directly in the line below

Reply:
ok